static/js/webrtc.js

```diff
@@ -314,6 +322,15 @@ var rtc = (function() {
         .append($largeVideo)
         .insertAfter($video);
     },
+    // Sends a stat to the back end. `statName` must be in the
+    // approved list on the server side.
+    sendErrorStat: function(statName) {
+      var msg = { "component" : "pad", // FIXME: Remove this stupidity!
+                  "type": "STATS",
```

Comment:
I saw `type: 'COLLABROOM', data{type='RTC_MESSAGE'}` in the other websocket message we have here. I'm not sure if I extrapolated that naming convention correctly or not.

Reply:
looks fine